src/test/java/org/opensearch/security/IndexIntegrationTests.java

```diff
@@ -846,19 +846,19 @@ public void testIndexResolveMinus() throws Exception {
         resc = rh.executeGetRequest("/*,-foo*/_search", encodeBasicHeader("foo_all", "nagilum"));
         assertThat(resc.getStatusCode(), is(HttpStatus.SC_FORBIDDEN));
 
-        resc = rh.executeGetRequest("/*,-*security/_search", encodeBasicHeader("foo_all", "nagilum"));
+        resc = rh.executeGetRequest("/*,-*security,-*resource*/_search", encodeBasicHeader("foo_all", "nagilum"));
```

Comment:
If the resource index is now hidden, can we roll back these changes?

Reply:
I tried but the tests fail without this change even though the index is marked as hidden which suggests that /* is resolving to hidden indices as well.
```json
{
  ".opensearch_resource_sharing" : {
    "settings" : {
      "index" : {
        "replication" : {
          "type" : "DOCUMENT"
        },
        "hidden" : "true",
        "number_of_shards" : "1",
        "auto_expand_replicas" : "0-all",
        "provided_name" : ".opensearch_resource_sharing",
        "creation_date" : "1741386115210",
        "number_of_replicas" : "0",
        "uuid" : "iDibox3zS8CNC0MhqrkTYQ",
        "version" : {
          "created" : "137217827"
        }
      }
    }
  }
}
```
As you can see the index is marked as hidden.

Reply:
Closing this comment. As it will get lost in the long github history. Please open a new comment to continue the discussion.